src/Spec2-Core/SpAbstractAdapter.class.st

```diff
@@ -195,6 +199,26 @@ SpAbstractAdapter >> sendRightClickEvent [
 	self subclassResponsibility
 ]
 
+{ #category : #releasing }
+SpAbstractAdapter >> unsubscribe [ 
```

Comment:
shouldn't this be in "private" ?

Reply:
I wanted something more meaningful... if `private` then `private-releasing` :-)

Reply:
with spaces `private - releasing`. Otherwise we will force Torsten to fix it ;)